hooks/useCreateSuggestionImage.ts

```diff
@@ -12,14 +14,13 @@ const useCreateSuggestionImage = () => {
   const [convertedKeyword, setConvertedKeyword] = useState<string>('')
   const [isError, setIsError] = useState(false)
   const [errorMessage, setErrorMessage] = useState('')
-  const [images, setImages] = useState<string[][] | File[][]>([])
+  const [images, setImages] = useState<ImagesTypes[]>([])
   const [isSendKeyword, setIsSendKeyword] = useState<boolean>(false)
 
   const createSuggestionImageMutation = useMutation(createSuggestionImage, {
```

Comment:
상기 리뷰와 이어지는 내용입니당
```suggestion
  const { mutate, error } = useMutation(createSuggestionImage, {
```
요런식으로 error 상태를 꺼내와 관리해도 괜찮지 않을까요??